draft-zheng-ccamp-client-tunnel-yang.md

```diff
@@ -0,0 +1,188 @@
+---
+coding: utf-8
+
+title: A YANG Data Model for Client-layer Tunnel
+abbrev: "Client Tunnel YANG Model"
+category: std
+
+docname: draft-zheng-ccamp-client-tunnel-yang-13
+submissiontype: IETF
+v: 3
+workgroup: CCAMP Working Group
+
+stand_alone: yes
+pi: [toc, sortrefs, symrefs, comments]
+
+author:
+  -
+    name: Chaode Yu
+    org: Huawei Technologies
+    email: yuchaode@huawei.com
+  -
+    name: Haomian Zheng
+    org: Huawei Technologies
+    street: H1, Huawei Xiliu Beipo Village, Songshan Lake
+    city: Dongguan
+    region: Guangdong
+    code: 523808
+    country: China
+    email: zhenghaomian@huawei.com
+  -
+    name: Aihua Guo
+    org: Futurewei
+    email: aihuaguo@futurewei.com
+  -
+    name: Italo Busi
+    org: Huawei Technologies
+    email: italo.busi@huawei.com
+  -
+    name: Yunbin Xu
+    org: CAICT
+    email: xuyunbin@caict.ac.cn
+  -
+    name: Yang Zhao
+    org: China Mobile
+    email: zhaoyangyjy@chinamobile.com
+  -
+    name: Xufeng Liu
+    org: Alef Edge
+    email: xufeng.liu.ietf@gmail.com
+
+contributor:
+  -
+    name: Yanlei Zheng
```

Comment:
Just a small issue, I remember that there used to be an security scanning issue in the inventory repository. shall we remove Chinese Operators' information on GitHub and add it manually when we submit the draft?

Reply:
See: https://github.com/italobusi/eth-te-tunnel/pull/1/commits/09770095de49de99f8f49b9ad5f35e7d33ee3998